README.md

```diff
@@ -87,7 +87,7 @@ git --version
    ```sh
    pip install -r fprime/requirements.txt
    ```
-2. Install the arduino-cli
+1. Install the arduino-cli
```

Comment:
It looks like your numbering got reset somehow. Recommend renumbering everything for QoL

Reply:
This is actually intentional. All the other numbers are 1 above this which was also intentional. Letting the markdown parser autonumbering do it's thing so as we adjust docs we don't have to go renumber everything by hand. Take a look at the preview and see it working! :-)

Reply:
Oh nice! I didn't realize that was possible.